documentation/how-to/digitize.en.md

```diff
@@ -139,6 +139,28 @@ A video demonstration of some editing possibilities:
 
 ![type:video](https://player.vimeo.com/video/499564854)<!-- markdown-link-check-disable-line -->
 
+## Erase Tool Functionality
+
+The erase tool is a new feature introduced in the geometry editor, designed to facilitate the removal of parts of a line or polygon geometry. This tool enables users to efficiently correct paths or polygons directly within the application interface and directly from the field.
+
+Suppose you have a polygon representing a building footprint, but there's an unwanted protrusion. You can use the erase tool to precisely remove that portion of the polygon, ensuring accuracy in your geometry representation.
```

Comment:
I'm not completely sure I'd call the erase tool "precise" :) The reshape tool is a much more precise tool that'd probably better fit this example.

I'd also note that the erase tool works best with a stylus/pen.

Reply:
Do you think is better to replace the gif, and also add this quote "for more precision utilize with a stylus/pen"?